modules/qna/src/backend/api.ts

```diff
@@ -51,7 +51,7 @@ export default async (bp: typeof sdk, botScopedStorage: Map<string, QnaStorage>)
     }
   })
 
-  router.put('/:question', async (req, res) => {
+  router.put('/questions/:question', async (req, res) => {
```

Comment:
I don't understand what's the question query param.. is it an ID (it should be)? if it is then please name it as id just like you did in the GET method for a matter of consistency. If it isn't then make it an ID please